site/docs/v1/tech/apis/scim/_scim-response-codes.adoc

```diff
@@ -14,21 +14,25 @@ ifdef::delete[]
 endif::[]
 
 |400
-|The request was invalid and/or malformed. The response will contain an link:/docs/v1/tech/apis/scim[SCIM Errors] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
+|The request was invalid and/or malformed. The response will contain a link:/docs/v1/tech/apis/scim[SCIM Error] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
 
+ifndef::unauthenticated[]
```

Comment:
`no_authorization` seems to be what is defined here: site/docs/v1/tech/apis/_standard-get-response-codes.adoc 

Please use that.

Reply:
done